features/consent/src/main/java/in/projecteka/jataayu/consent/ui/fragment/ConsentListFragment.kt

```diff
@@ -214,19 +212,10 @@ class GrantedFragment : BaseFragment(), AdapterView.OnItemSelectedListener,
         (rvConsents.adapter as ConsentsListAdapter).updateData(requests)
     }
 
-    private fun unregisterEventBus() {
-        if (EventBus.getDefault().isRegistered(this)) {
-            EventBus.getDefault().unregister(this)
-        }
-    }
-
     override fun askForConsentPin(iDataBindingModel: IDataBindingModel) {
-        if (!EventBus.getDefault().isRegistered(this)) {
-            EventBus.getDefault().register(this)
-        }
         consentToRevoke = (iDataBindingModel as Consent)
         val intent = Intent(context, PinVerificationActivity::class.java)
-        startActivity(intent)
+        startActivityForResult(intent, 701)
```

Comment:
add the constant to companion object. if required, use the value from the companion object of the other activity